src/many-abci/src/module.rs

```diff
@@ -230,14 +321,23 @@ impl<C: Client + Send + Sync> blockchain::BlockchainModuleBackend for AbciBlockc
         })?;
 
         if let Some(block) = block {
-            let block = _many_block_from_tendermint_block(block);
+            let block = _many_block_from_tendermint_block(
+                block,
+                blockchain::ListArgs {
+                    count: None,
+                    order: None,
+                    filter: None,
+                },
```

Comment:
I don't think you need to compute it every time.

Since you're hardcoding the parameters, it will always return the same result.

Reply:
@fmorency There is another instance of use of `_many_block_from_tendermint_block` where I am _not_ hardcoding.  Also, in this case, where none of these configs are in scope, what should I place here?